app/services/call_creator.rb

```diff
@@ -92,6 +96,11 @@ def place_call
     end
   end
 
+  def publish_event
+    return if @call.member.blank? # handle this in worker
```

Comment:
AK doesn't support creating actions for anonymous users. Champaign call tool does support that. I believe this check should be done in the worker, and not here. That's why there's a comment. I'll try to get that in today.